draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -983,6 +1004,16 @@ when presenting to a downstream party.
 In some scenarios this behavior could be desirable,
 but if it is not, Issuers need to support and Verifiers need to enforce Key Binding.
 
+## Integrity of Presentation
+
+In a Presentation, the Issuer-signed JWT is integrity-protected by the Issuer's
+signature, and the Disclosures are integrity-protected by the digests included
+in the Issuer-signed JWT. However, without enforcing a Key Binding JWT as
+defined in (#kb-jwt), an attacker that can intercept the Presentation can remove
+Disclosures. The attacker could in theory also add Disclosures, but that would
+require the attacker to know the salt and claim value for the claim to be added,
+which is infeasible if the salt has sufficient entropy.
```

Comment:
I am sorry for getting hung up on this one but I do worry that talking about "an attacker that can intercept the Presentation" here might cause a lot of questions/consternation/concern from readers and reviewers. So here's an attempt at saying more or less the same thing but with fewer words and no specific mention of an attacker. 

```suggestion
in the Issuer-signed JWT. However, without enforcing a Key Binding JWT as
defined in (#kb-jwt), the integrity of the selected set of Disclosures the 
Holder elected to send is not ensured.
```

Reply:
elected or selected?

Reply:
The word "selected" was already in the sentence earlier so I went with "elected" for word choice variety. But "selected" would be fine, if you think it's better/clearer?

Reply:
or ... change so that "selected" is used just once? 

```suggestion
in the Issuer-signed JWT. However, without enforcing a Key Binding JWT as
defined in (#kb-jwt), the integrity of the set of Disclosures the 
Holder selected to send is not ensured.
```